package.json

```diff
@@ -63,13 +63,13 @@
     "@types/node": "^7.0.16",
     "@types/resolve": "^0.0.4",
     "@types/semver": "^5.3.30",
-    "chai": "^3.5.0",
-    "github": "^8.2.1",
+    "chai": "^4.0.1",
+    "github": "^9.2.0",
```

Comment:
the new `github` API will break the `generate-changelog` script. I tried that last week or so